data_interrogator/aggregators.py

```diff
@@ -0,0 +1,127 @@
+import django.db.models 
+import django.db.models.functions
+from django.db.models import functions as func
+from django.db.models import Count, Min, Max, Sum, Value, Avg
+from data_interrogator.db import GroupConcat, DateDiff, ForceDate, SumIf, ComplexLookup
+from data_interrogator import exceptions as di_exceptions
+
+from data_interrogator.utils import normalise_field, normalise_math
+
+
+aggregate_register = {}
+def register_aggregator(Aggregate):
+    if Aggregate.command in aggregate_register.keys():
+        raise di_exceptions.DuplicateAnnotationCommand(Aggregate, aggregate_register[Aggregate.command])
+    else:
+        aggregate_register[Aggregate.command] = Aggregate
+    return Aggregate
+
+
+class InterrogatorFunction:
+    def process_arguments(self, argument_string):
+        return [argument_string], {"distinct": False}
```

Comment:
Why always {'distinct': False}?

Reply:
This has been retained from the original code, during continued major refactor we can investigate the impact of changing the default.